src/hyperlight_host/examples/tracing-tracy/main.rs

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Is it worth adding this example to CI?

Reply:
Hmm I don't think so, we don't run all examples currently. Maybe something to consider in the future.